modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js

```diff
@@ -315,7 +315,7 @@ $(document).ready(function () {
             $(errorMsg).text("Notifier frequency is a required field. It cannot be empty.");
             $(errorMsgWrapper).removeClass("hidden");
         } else if (notifierType == notifierTypeConstants["LOCAL"] && !isPositiveInteger(notifierFrequency)) {
-            $(errorMsg).text("Provided notifier frequency is invalid. Please check.");
+            $(errorMsg).text("Provided notifier frequency is invalid. ");
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158